lib/metricsImporter/cli.js

```diff
@@ -0,0 +1,224 @@
+const client = require('./client');
+
+function validateOne(option, value) {
+  if (!value) {
+    console.error(`--${ option } is required`);
+    return false;
+  }
+  if (Array.isArray(value)) {
+    console.Error(`Specify only one -${ option }`);
```

Comment:
what's up with `console.error` VS `console.Error` ? I'd also be consistent in the general syntax of the error message. Maybe "--${ option } must have exactly one value"  ?

Reply:
It's a typo.